README.md

```diff
@@ -1 +1,235 @@
-# IoT-STH
+#<a id="section0"></a> IoT-STH
+
+* [Introduction] (#section1)
+* [Dependencies](#section2)
+* [Installation](#section3)
+* [Inserting data (single events and aggregated data) into the database](#section4)
+* [Querying data from the database](#section5)
+* [Running the STH server](#section6)
+* [Contact](#section7)
+
+##<a id="section1"></a> Introduction
+The STH component is a FIWARE component in charge of providing aggregated time series information about the evolution of
+entity attribute values in time.
+
+The aggregated time series information is stored in a MongoDB instance. This information can be generated by 2 main
+means:
+
+1. The STH component can directly subscribe to the Context Broker to receive notifications when the entity attribute
+values change, calculating the aggregated time series information and storing it in the MongoDB instance.
+This option is called the minimalist option.
+2. A new sink can be enabled in the Cygnus component to calculate and to update the aggregated time series information
+as the entity attribute values change over time. This option is the formal one.
+
+Regarding the aggregated time series information provided by the STH component, there are 4 main concepts which are
+important to know about:
+
+* <b>Range</b>: The period of time about which the aggregated time series information is provided. Possible valid
+ranges values are: year, month, day, hour, minute.
+* <b>Resolution</b> or <b>aggregation period</b>: The time period by which the aggregated time series information is grouped.
```

Comment:
Yeah, I included it for an easier reading in an IDE :) Do you prefer avoiding this carriage returns? :)